### ISSUE TITLE
Question about learning rate of centrers

### ISSUE THREAD

**Issue by hoangcuong2011** [shamangary/Keras-MNIST-center-loss-with-visualization#7]
Hello,

Thanks for your code. I just have a very quick question.

The paper mentions that there should be a scalar alpha to control the learning rate of the centers. This is interesting to me. However I don't think you implement such a thing. If you do please correct me.
Plus do you know how to implement that? I am very curious about this but right now I don't have a concrete idea about doing so.
Thx

**Comment by zhangxiann** [comment]
I have the same question about the learning rate of centrers...
